lib/ansible/modules/cloud/azure/azure_rm_functionapp.py

```diff
@@ -25,10 +25,28 @@
         description:
             - Name of resource group
         required: true
+        aliases:
+            - resource_group_name
     name:
         description:
             - Name of the Azure Function App
         required: true
+    location:
+        description:
+            - Valid Azure location. Defaults to location of the resource group.
+        version_added: 2.6
+    storage_account:
+        description:
+            - Name of the storage account to use.
+        required: true
+        aliases:
+            - storage
+            - storage_account_name
+        version_added: 2.6
```

Comment:
@zikalino The value of `version_added` should always be the release in which the feature was added. If the feature was added when the module was created it should not be defined.

Setting this to the correct value will cause a CI error if `version_added` does not match the current release, but we can bypass that as CI doesn't understand options which were not originally documented.

Can you open a PR to correct this and any other of the `version_added` entries?